Registry/Registry.EM_COMMON

```diff
@@ -1781,7 +1806,7 @@ state    real  CKA              ij      misc        1         -      r        "C
 state    real  CD               ij      misc        1         -      r        "CD"                    "DRAG COEFF AT 10m"                      ""
 state    real  CDA              ij      misc        1         -      r        "CDA"                   "DRAG COEFF AT LOWEST MODEL LVL"         ""
 state    real  UST              ij      misc        1         -      rh       "UST"                   "U* IN SIMILARITY THEORY"                      "m s-1"      
-state    real  USTM             ij      misc        1         -      r        "USTM"                  "U* IN SIMILARITY THEORY WITHOUT VCONV"        "m s-1"
+state    real  USTM             ij      misc        1         -      rh       "USTM"                  "U* IN SIMILARITY THEORY WITHOUT VCONV"        "m s-1"
```

Comment:
@zhangx-3dtke Do you need this variable to be in history file?

Reply:
No, I output this variable for diagnosis and forgot to change back.

Reply:
Restart run test passed. I added it to "Tests Conducted" @weiwangncar